exporter/awskinesisexporter/internal/producer/batcher.go

```diff
@@ -18,6 +18,7 @@ import (
 
 type batcher struct {
 	stream *string
+	arn    *string
```

Comment:
Should these be used together or should they be seperate?

Reply:
According to the AWS document, both ways are valid:


>When invoking this API, you must use either the StreamARN or the StreamName parameter, or both. It is recommended that you use the StreamARN input parameter when you invoke this API.

ref: https://docs.aws.amazon.com/kinesis/latest/APIReference/API_DescribeStream.html